packages/app/src/app/pages/Sandbox/Editor/Workspace/index.js

```diff
@@ -16,8 +16,8 @@ import Deployment from './items/Deployment';
 import ConfigurationFiles from './items/ConfigurationFiles';
 import NotOwnedSandboxInfo from './items/NotOwnedSandboxInfo';
 
-import ConnectionNotice from './ConnectionNotice';
-import Advertisement from './Advertisement';
+import { ConnectionNotice } from './ConnectionNotice';
```

Comment:
This is really nice!

Reply:
So keep in mind that with my previous comment, I'm in the process of re-organizing files one by one to keep these reviews easy on you. Every component will be refactored to conform to the conventions I'm looking to establish here.